code/modules/spells/spell_types/pointed/mind_transfer.dm

```diff
@@ -39,6 +39,16 @@
 		if(stand.summoner)
 			victim = stand.summoner
 
+	//You should not be able to enter one of the most powerful side-antags as a fucking wizard.
+	if(istype(victim,/mob/living/simple_animal/slaughter))
+		to_chat(user, "<span class='warning'>The devilish contract doesn't include the 'mind swappable' package, please try again another lifetime.</span>")

```

Comment:
what if you didn't summon this slaughter demon? they can technically spawn on their own as an extremely rare midround antag

Reply:
you still shouldnt be able to enter the demon